assets/src/console/pages/products/ListPage.tsx

```diff
@@ -56,35 +87,38 @@ const ListPage: React.FC<props> = ({ service, gatewayAccount }) => {
             {flatMap(
               gatewayAccount => gatewayAccount.products,
               data.service.gatewayAccounts
-            ).length === 0 && (
+            ).length === 0 ? (
               <Warning>
                 <P>You don’t have any payment links.</P>
               </Warning>
+            ) : (
+              <>
+                <Ul>
+                  {flatMap(
```

Comment:
it seems `flatMap` is not available in Edge nor IE so we will need to include a polyfill for this, no? ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap

Reply:
it's defined in this file at line 20

Reply:
oh right i see. assumed it was just using the window/global version. any reason we can't use the global version and use the `core-js` polyfill in index.ts with our other polyfill imports?